### ISSUE TITLE
VR ScriptEvents missing TESFastTravelEndEvent 

### ISSUE THREAD

**Issue by alandtse** [CharmedBaryon/CommonLibSSE-NG#49]
FYI, just determined that VR doesn't have TESFastTravelEndEvent 
https://github.com/alandtse/CommonLibVR/commit/ed163dc976caaa20e51cd631e662bcc8d9d440cd

**Comment by jpstewart** [comment]
Will have to investigate any implications. Generally I leave all classes available all the time, since someone may want to have a plugin that uses a feature available in one runtime if and only if it is being used in that runtime. Most likely this means altering `ScriptEventSourceHolder` to have a getter to test if it is a valid event source for the event.

**Comment by jpstewart** [comment]
Probably something like

```c++
[[nodiscard]] inline RE::BSTEventSource<TESFastTravelEndEvent>* AsTESFastTravelEndEventSource() const noexcept
{
[[nodiscard]] WorldSpaceMenu* AsWorldSpaceMenu() noexcept
{
    if SKYRIM_REL_CONSTEXPR (REL::Module::IsVR()) {
        return nullptr;
    }
    return &REL::RelocateMember<BSTEventSource<TESFastTravelEndEvent>>(this, 0x1238, 0);
}
```

**Comment by jpstewart** [comment]
Fixed by `eda126c91b0f0b7d20d55ac26081a848403f8af2`.
